listenbrainz/spark/request_manage.py

```diff
@@ -165,12 +165,15 @@ def request_candidate_sets(days, top, similar):
 @cli.command(name='request_recommendations')
 @click.option("--top", type=int, default=200, help="Generate given number of top artist recommendations")
 @click.option("--similar", type=int, default=200, help="Generate given number of similar artist recommendations")
-def request_recommendations(top, similar):
+@click.option("--mb_id", callback=parse_list, default=[], multiple=True, help="Generate recommendations for given users" \
```

Comment:
I think user-ids might be better; also note that options should use - and not _ to separate words. Also, how are more than one id specified? comma seperated? I think the usage statement should indicate this.

Reply:
And this comment really applies to this whole PR...

Reply:
I guess the usage should be updates in the Readme like it is for other commands.

Reply:
we have been using user id on server side for some integer. I guess it will be confusing to use user id here. 

We actually want to send musicbrainz ids of users to spark.